### PR TITLE
use 'kvm-bindings' from 'firecracker-microvm'

### PR DESCRIPTION
**Pull request by acatangiu**
*Issue #, if available:*

Part of https://github.com/firecracker-microvm/firecracker/issues/1831

*Description of changes:*

Consume 'kvm-bindings' dependency from 'github.com/firecracker-microvm' which provides versioned bindings for x86_64 state structs.

Release internal version `v0.5.0-1`.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by sandreim — DISMISSED** [review]

File: coverage_config_aarch64.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "coverage_score": 76.2,
+  "coverage_score": 76.7,
```
Comment (andreeaflorescu) [inline coverage_config_aarch64.json:2]: Hmm, why is this changed? Shouldn't it be changed upstream as well?

**Review by sandreim — APPROVED** [review]

File: coverage_config_aarch64.json (hunk 1 above)
Comment (acatangiu) [reply coverage_config_aarch64.json:2]: I don't know, could it be because of different kvm-bindings?

You just changed it upstream and no other changes went in in the meantime.

**Review by andreeaflorescu — APPROVED** [review]
